### DISCUSSION TITLE
Shifting moveit planning_scene

### DISCUSSION THREAD

**Discussion by Sushant-Chavan** [hsr-project/general#15] [Q&A]
Hello,
In our simulations, we setup the moveit planning_scene for the objects in the environment. However, when the robot moves around in the environment, we notice that the planning scene starts shift significantly, especially when the robot rotates This can be seen in the below issue. 

https://user-images.githubusercontent.com/9501294/122648807-17524a80-d12b-11eb-8992-811242185418.mp4

Does anyone know what could be causing this? Or how to fix this issue?

**Comment by yosuke** [comment]
It looks like the planning scene is using `odom` frame as a frame_id.
Changing to `map` frame may fix the problem (or reverse).
